test/junit/scala/tools/nsc/backend/jvm/analysis/NullnessAnalyzerTest.scala

```diff
@@ -217,4 +217,7 @@ class NullnessAnalyzerTest extends ClearAfterClass {
       (trim, 3, NotNull)  // receiver at `trim`
     )) testNullness(a, m, insn, index, nullness)
   }
+
+  // add "sm" interpolator
+  implicit class StrContextStripMarginOps(val stringContext: StringContext) extends scala.reflect.internal.util.StripMarginInterpolator
```

Comment:
There is a ready made implicit class importable from `scala.reflect.internal.util.StringContextStripMarginOps`.

scala> import scala.reflect.internal.util.StringContextStripMarginOps
import scala.reflect.internal.util.StringContextStripMarginOps

```
scala> sm"foo"
res0: String = foo
```